### ISSUE TITLE
Missing "zstd" Python package for working with Zstandard compression

### ISSUE THREAD

**Issue by Isaakkamau** [mozilla/bugbug#3708]
When training the bug model I encountered this error
```(venv) PS C:\Users\Isaac\bugbug> python -m scripts.trainer bug
2023-10-11 22:23:50,858:INFO:bugbug.db:Downloading https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.data_bugs.latest/artifacts/public/bugs.json.zst to data/bugs.json.zst
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Isaac\bugbug\scripts\trainer.py", line 161, in <module>
    main()
  File "C:\Users\Isaac\bugbug\scripts\trainer.py", line 157, in main
    assert db.download(required_db)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Isaac\bugbug\bugbug\db.py", line 99, in download
    utils.extract_file(zst_path)
  File "C:\Users\Isaac\bugbug\bugbug\utils.py", line 291, in extract_file
    zstd_decompress(inner_path)
  File "C:\Users\Isaac\bugbug\bugbug\utils.py", line 247, in zstd_decompress
    subprocess.run(["zstd", "-df", f"{path}.zst"], check=True)
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\subprocess.py", line 1024, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Python311\Lib\subprocess.py", line 1509, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Comment by Isaakkamau** [comment]
@suhaibmujahid Please check on this issue, Can I open a PR for it?

**Comment by suhaibmujahid** [comment]
@Isaakkamau Could you please elaborate on the cause of this issue?

**Comment by Isaakkamau** [comment]
@suhaibmujahid I have cloned the project created a virtual environment and installed all requirements.txt but I am getting a lot of errors when trying to train the models. I have already solved a couple of them and I'm requesting if I can open issues about the errors and create PR for them?

For the above error it appeared when I was training the bug model, here is the command I used: `(venv) PS C:\Users\Isaac\bugbug> python -m scripts.trainer bug`

I solved it by adding `zstd==1.5.5.1` as a requirement.

Thank you

**Comment by suhaibmujahid** [comment]
>  I have already solved a couple of them and I'm requesting if I can open issues about the errors and create PR for them?

Feel free to open a PR. Please consider solving only one problem in the same PR. You could open multiple PRs if necessary.
